ui/app/AppLayouts/Wallet/views/CollectiblesView.qml

```diff
@@ -58,121 +59,165 @@ StatusScrollView {
-                        return d.controller.lessThan(modelLeft.symbol, modelRight.symbol)
-                    }
-                    enabled: d.isCustomView
-                }
-            ]
-        }
-
-        readonly property var communityCollectiblesModel: SortFilterProxyModel {
-            sourceModel: d.renamedModel
-            filters: [
-                ExpressionFilter {
-                    expression: {
-                        d.controller.settingsDirty
-                        return d.symbolIsVisible(model.symbol) && !!model.communityId
-                    }
-                }
-                // TODO add other sort/filter using ManageTokensController (#12517)
-            ]
-            sorters: [
-                RoleSorter {
-                    roleName: "name"
-                    enabled: !d.isCustomView
-                },
-                ExpressionSorter {
-                    expression: {
-                        d.controller.settingsDirty
-                        return d.controller.lessThan(modelLeft.symbol, modelRight.symbol)
-                    }
-                    enabled: d.isCustomView
-                }
-            ]
-        }
-
-        readonly property bool hasCollectibles: d.regularCollectiblesModel.count
-        readonly property bool hasCommunityCollectibles: d.communityCollectiblesModel.count
+        readonly property bool hasCollectibles: regularCollectiblesView.count
+        readonly property bool hasCommunityCollectibles: communityCollectiblesView.count
 
         readonly property var controller: ManageTokensController {
             settingsKey: "WalletCollectibles"
         }
 
         function hideAllCommunityTokens(communityId) {
-            const tokenSymbols = ModelUtils.getAll(communityCollectiblesModel, "symbol", "communityId", communityId)
+            const tokenSymbols = ModelUtils.getAll(communityCollectiblesView.model, "symbol", "communityId", communityId)
             d.controller.settingsHideCommunityTokens(communityId, tokenSymbols)
         }
     }
 
-    ColumnLayout {
-        width: root.availableWidth
-        spacing: 0
+    component CustomSFPM: SortFilterProxyModel {
+        id: customFilter
+        property bool isCommunity
 
-        ShapeRectangle {
-            visible: !d.hasCollectibles && !d.hasCommunityCollectibles
-            Layout.fillWidth: true
-            text: qsTr("Collectibles will appear here")
+        sourceModel: d.renamedModel
+        proxyRoles: JoinRole {
+            name: "groupName"
+            roleNames: ["collectionName", "communityName"]
         }
+        filters: [
+            ExpressionFilter {
+                expression: {
+                    d.controller.settingsDirty
+                    return d.controller.filterAcceptsSymbol(model.symbol) && (customFilter.isCommunity ? !!model.communityId : !model.communityId)
+                }
+            }
+        ]
+        sorters: [
```

Comment:
Yeah that's the ideal case we want to achieve in the future; for now this is how it works and we really can't do anything better about it

Reply:
But are we missing backend for these? Bc pagination is already in place, so probably it's matter of changing the data request criteria? @dlipicar

Reply:
There's no requests to be emitted, it's one of the ways how a regular QAIM works; this would have to be implemented

Reply:
I don't know if pagination for collectibles is ready or not, it's question to @dlipicar probably, as @noeliaSD mentioned. 

However, I think that we could define some reasonable api on the store for setting criteria and taking collectibles according to that criteria. Within a store we could temporarily simulate desired sorting/filtering with sfpm. But it would be just a single point for change when backend is ready. 

At this point where we have clearer the requirements and final design, I don't see more benefits on having a temporal solution. So I would opt of focusing on the architecture we aim to have and, cover on this pr the main assumptions and organize the UI and leave it ready for backend integration. So only final adjustments will need to be done in one place (the store).

Currently it's really easy to loose track what's target solution and what is temporary so imo we can start reaching the final solution already in here.

Reply:
This SFPM is relying on the custom C++ controller to get the correct "Custom order", for filtering out the explicitly hidden items and also the SortCombo to get the other predefined sort parameters. With this sorting combo (and soon the filtering combo), it's truly a very custom view. I can't see how we could generalize this by putting some stuff into a Store, or how it could be helpful for us in the long term (having in mind the new design and backend architecture)

Reply:
We don't have sorting criteria added to our collectibles API yet. I could quickly add the simple ones (by name for example), the custom one will take me a bit longer.
Whenever we start using those, we should remove sorting for collectibles in the client (since it really doesn't look good to sort on unordered incomplete data, and if the data returned by the backend is already sorted then there's nothing else to do here).

I'm gonna work on this next week!

Reply:
Sure no prob, we can nerf the local sorting here whenever the backend has support for it

Reply:
Ok, let's then do another iteration on it. Not necessary to have the backend ready so we can prepare the UI since we know/imagine/propose what will be the api interaction with the backend!